src/fabric/ScreenStackHeaderConfigNativeComponent.ts

```diff
@@ -50,5 +50,7 @@ export interface NativeProps extends ViewProps {
 
 export default codegenNativeComponent<NativeProps>(
   'RNSScreenStackHeaderConfig',
-  {}
+  {
+    interfaceOnly: true,
+  }
```

Comment:
Why do we need this?

Reply:
So that we can have custom descriptor for header config, otherwise (simplifying here a bit) RN will use the autogenerated one.